packages/inter-protocol/test/price/test-fluxAggregator.contract.js

```diff
@@ -0,0 +1,794 @@
+    { message: 'round not accepting submissions' },
+  );
+  await E(pricePushAdminB).pushPrice({ roundId: 2, unitPrice: 2000n });
+  await E(pricePushAdminC).pushPrice({ roundId: 2, unitPrice: 3000n });
+  await oracleTimer.tick();
+
+  const round1Attempt2 = await E(aggregator.creatorFacet).getRoundData(1);
+  t.is(round1Attempt2.answer, 200n);
+  const round2Attempt1 = await E(aggregator.creatorFacet).getRoundData(2);
+  t.is(round2Attempt1.answer, 2500n);
+
+  t.log('----- round 3: check oracle submission order');
+  // unlike the previous test, if C initializes, all submissions should be recorded,
+  // which means the median will be the expected 5000 here
+  await oracleTimer.tick();
+  await E(pricePushAdminC).pushPrice({ roundId: 3, unitPrice: 5000n });
+  await E(pricePushAdminA).pushPrice({ roundId: 3, unitPrice: 4000n });
+  await E(pricePushAdminB).pushPrice({ roundId: 3, unitPrice: 6000n });
+  await oracleTimer.tick();
+
+  const round1Attempt3 = await E(aggregator.creatorFacet).getRoundData(1);
+  t.is(round1Attempt3.answer, 200n);
+  const round3Attempt1 = await E(aggregator.creatorFacet).getRoundData(3);
+  t.is(round3Attempt1.answer, 5000n);
+});
+
+test('timeout', async t => {
+  const { zoe } = t.context;
+
+  const aggregator = await t.context.makeChainlinkAggregator({
+    ...defaultConfig,
+    restartDelay: 2,
+    timeout: 5,
+  });
+  /** @type {{ timer: ManualTimer }} */
+  // @ts-expect-error cast
+  const { timer: oracleTimer } = await E(zoe).getTerms(aggregator.instance);
+
+  const pricePushAdminA = await E(aggregator.creatorFacet).initOracle(
+    'agorice1priceOracleA',
+  );
+  const pricePushAdminB = await E(aggregator.creatorFacet).initOracle(
+    'agorice1priceOracleB',
+  );
+  const pricePushAdminC = await E(aggregator.creatorFacet).initOracle(
+    'agorice1priceOracleC',
+  );
+
+  // ----- round 1: basic consensus w/ ticking: should work EXACTLY the same
+  await oracleTimer.tick();
+  await E(pricePushAdminA).pushPrice({ roundId: 1, unitPrice: 100n });
+  await oracleTimer.tick();
+  await E(pricePushAdminB).pushPrice({ roundId: 1, unitPrice: 200n });
+  await oracleTimer.tick();
+  await E(pricePushAdminC).pushPrice({ roundId: 1, unitPrice: 300n });
+
+  const round1Attempt1 = await E(aggregator.creatorFacet).getRoundData(1);
+  t.is(round1Attempt1.roundId, 1n);
+  t.is(round1Attempt1.answer, 200n);
+
+  // ----- round 2: check restartDelay implementation
+  // timeout behavior is, if more ticks pass than the timeout param (5 here), the round is
+  // considered "timedOut," at which point, the values are simply copied from the previous round
+  await oracleTimer.tick();
+  await E(pricePushAdminB).pushPrice({ roundId: 2, unitPrice: 2000n });
+  await oracleTimer.tick();
+  await oracleTimer.tick();
+  await oracleTimer.tick();
+  await oracleTimer.tick();
+  await oracleTimer.tick();
+  await oracleTimer.tick(); // --- should time out here
```

Comment:
`tickN(6)` would be better and is supposed to do the same thing. I recently noticed that it doesn't seem to be equivalent. Can you see why it makes a difference?

Reply:
This file was redundant with `test-fluxAggregator` and I removed it in a subsequent PR. I've rebased to remove it here.